### DISCUSSION TITLE
Stopping an instance

### DISCUSSION THREAD

**Discussion by SundaramMaheshkumar** [bpmnServer/bpmn-server#82] [Q&A] [answered]
Hi @ralphhanna,

Is there a way to stop (not delete) a running instance using `bpmn-client` ?

Thanks.

**Comment by ralphhanna** [comment, marked as answer]
Hi,

Yes, by creating a catch-signal event in the process leading to an End Event, Whenever you want to cancel issue signal from client

Thanks
________________________________
From: Maheshkumar Sundaram ***@***.***>
Sent: Wednesday, April 5, 2023 8:32 AM
To: ralphhanna/bpmn-server ***@***.***>
Cc: ralphhanna ***@***.***>; Mention ***@***.***>
Subject: [ralphhanna/bpmn-server] Stopping an instance (Discussion #82)


Hi @ralphhanna<https://github.com/ralphhanna>,

Is there a way to stop a running instance using bpmn-client ?

Thanks.

**Reply by SundaramMaheshkumar** [reply]
Thank you
